vnext/ReactUWP/Views/FrameworkElementViewManager.cpp

```diff
@@ -385,6 +385,36 @@ void FrameworkElementViewManager::StartTransformAnimation(
   uielement.StartAnimation(expression);
 }
 
+folly::dynamic FrameworkElementViewManager::GetCommands() const
+{
+  // Don't update SetFocus/Blur commands in subclass, otherwise commands may not match with what defined in js side.
+  auto commands = Super::GetCommands();
+  commands.update(folly::dynamic::object
+  ("SetFocus", static_cast<std::underlying_type_t<FocusCommand>>(FocusCommand::SetFocus))
+  );
+  commands.update(folly::dynamic::object
+  ("Blur", static_cast<std::underlying_type_t<FocusCommand>>(FocusCommand::Blur))
+  );
+  return commands;
+}
+
+void FrameworkElementViewManager::DispatchCommand(XamlView viewToUpdate, int64_t commandId, const folly::dynamic& commandArgs)
+{
+  // UWP doesn't have the blur concept. Here we use FocusState::Pointer to move away to keyboard focused visual.
+  if (viewToUpdate)
```

Comment:
>if (viewToUpdate) [](start = 2, length = 17)

Why the nullptr check?  can this be an assert instead?

Reply:
agree assert would be better. I didn't see any reason why not assert